symplyphysics/laws/electricity/electromotive_force_of_induction_in_rotating_ce_of_induction_in_rotating_circuit.py

```diff
@@ -0,0 +1,52 @@
+from sympy import (Eq, solve, sin)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless, angle_type)
+
+# Description
+## With uniform rotation of a multi-turn frame with the same areas of turns in a uniform magnetic field,
+## an induction EMF occurs in each turn of this frame, so that the total EMF is numerically equal to the
+## product of the number of turns of the frame N, the induction of the magnetic field, the area of one
+## turn of the frame, the angular velocity of rotation of the frame and the sine of the angle of rotation
+## of the frame.
+## EMF - electromotive force.
+
+## Law is: U = -N * B * S * w * sin(w * t), where
```

Comment:
Shouldn't it also depend from initial phase?

Reply:
> Shouldn't it also depend from initial phase?

In general, nothing prevents you from adding some initial phase. Then it will mean that at the initial moment of time the frame was already deflected so that its normal was not parallel to the field induction lines. But I have not found this formula anywhere in this form (with an initial phase).